src/txn/txn.c

```diff
@@ -1663,6 +1663,9 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
                  */
                 if (!F_ISSET(op, WT_TXN_OP_KEY_REPEATED))
                     WT_ERR(__txn_resolve_prepared_op(session, op, true, &cursor));
+
+                /* Sleep between resolving prepared operations when configured. */
+                __wt_timing_stress(session, WT_TIMING_STRESS_PREPARE_RESOLUTION, NULL);
```

Comment:
Just to confirm my understanding, this timing stress doesn't generate this failure as we do not release the pinned page at this location. But this change is useful to simulate longer prepare resolution.

Reply:
It's not going to find this bug, as this bug is now fixed but it could find bugs like it in the future.